src/Tickets/Commerce/Gateways/PayPal/REST/Order_Endpoint.php

```diff
@@ -70,7 +70,7 @@ public function register() {
 
 		register_rest_route(
 			$namespace,
-			$this->get_endpoint_path() . '/(?P<order_id>[0-9a-zA-Z]+)',
+			$this->get_endpoint_path() . '/(?P<order_id>[0-9a-zA-Z]+)(?:/(?P<payer_id>[0-9a-zA-Z]+))?',
```

Comment:
We dont need the payer ID to be part of the URL.
It can be a param in the body.